app/javascript/src/Stats/lib/metrics_selector.jsx

```diff
@@ -44,17 +51,23 @@ export class StatsMetricsSelector extends StatsUI {
     this._setState({selectedMetricName: metric.systemName}, ['redraw'])
   }
 
+  update (metrics) {
+    this.metrics = metrics
```

Comment:
Shouldn't this be `this.metricsList`? Because of change in [line 11](https://github.com/3scale/porta/pull/575/files#diff-db9069c26b38a3a7dd1b76258ce850d5R11).

Reply:
Using a getter/setter always have a minor impact when refactoring. Also gives you the opportunity to expand, in this case, with the `update` method. It also makes a uniform interface for similar classes and concepts. Likewise, you could name the private properties differently and more specific on what they are storing where external objects don't necessarily need to know.